presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -1221,15 +1220,15 @@ public void testOrderByWithAggregation()
                 .setSystemProperty(LEGACY_ORDER_BY, "true")
                 .build();
 
-        for (Session session : asList(getSession(), legacyOrderBy)) {
-            for (String groupBy : asList("x.letter", "letter")) {
-                for (String orderBy : asList("x.letter", "letter")) {
-                    for (String output : asList("", ", letter", ", letter as y")) {
-                        assertQueryOrdered(session, format("select count(*) %s from (select substr(name,1,1) letter from nation) x group by %s order by %s", output, groupBy, orderBy));
-                    }
-                }
-            }
-        }
+        queryTemplate("select count(*) %output% from (select substr(name,1,1) letter from nation) x group by %groupBy% order by %orderBy%")
```

Comment:
😕 

(i'm biased.. i'm the author of the `for`-based code here)

Reply:
ah... I am so sorry. What can I do, I am the author of `queryTemplate` ;)